### DISCUSSION TITLE
Can't display svg in README.md

### DISCUSSION THREAD

**Discussion by Whiskium** [community/community#118201] [Mobile]
### Select Topic Area

Bug

### Body

I add a official workflow status badge (SVG) in the README.md. It's well displayed on the website, but NOT on the mobile app, neither on Android nor iOS, neither \<img\> nor "\!\[\]\(\)".

![1712553700293](https://github.com/community/community/assets/48499630/de15644d-9e5f-4117-8486-544e87e3f6e8)

I've read several previous discussions and it seems that this problem is not only happening to me. But I can't find any solution yet.

**Comment by warioddly** [comment]
This is probably a bug in SVG, in fact it should have worked on IMG or ![]() tag

Example: 
`![ALT](URL)`
or 
`<img src="SVG_URL"/>`

**Reply by Whiskium** [reply]
But it's provided by GitHub and well displayed on the website (Desktop & Mobile).

**Reply by warioddly** [reply]
Can you send your SVG to me

**Reply by v1dya-sagar** [reply]
You can refer to [this](https://stackoverflow.com/questions/13808020/include-an-svg-hosted-on-github-in-markdown) discussion on Stack Overflow

**Reply by Whiskium** [reply]
> Can you send your SVG to me

###

Here is the link. I may find the reason just now.

This SVG file is the status badge of a workflow of a PRIVATE repo.

When I add this badge of this private repo into the README, it can be displayed on the website but not on the mobile.
When I add a badge of a public repo into the README, it can be well displayed on both website and mobile.

Now the problem is not that all SVG can't be displayed on mobile, but the SVG badge of a private repo can be displayed on the website but can't on the mobile app.

**Reply by Whiskium** [reply]
> You can refer to [this](https://stackoverflow.com/questions/13808020/include-an-svg-hosted-on-github-in-markdown) discussion on Stack Overflow

I just find the reason above. Thank you for your help.

**Reply by Whiskium** [reply]
Is this a bug? Since the same thing behaves differently on the browser and app. It's expected to display but not.
